internals/templates/containers/LanguageProvider/index.js

```diff
@@ -17,7 +17,7 @@ import { makeSelectLocale } from './selectors';
 export class LanguageProvider extends React.PureComponent { // eslint-disable-line react/prefer-stateless-function
   render() {
     return (
-      <IntlProvider locale={this.props.locale} key={this.props.locale} messages={this.props.messages[this.props.locale]}>
+      <IntlProvider locale={this.props.locale} key={this.props.locale} messages={this.props.messages[this.props.locale]} textComponent={React.Fragment}>
```

Comment:
What's the benefit of using React.Fragment instead of just 
```jsx
<IntlProvider textComponent={children => children} />
```

Reply:
Since we sometimes want `children` to only render strings, if you passed a string into `this.props.children`, like "Hi I'm a string" and then tried to render it as a React Component   object inside `textComponent={this.props.children}`, it would be similar to doing `<Hi I'm a string>Hi I'm a string</Hi I'm a string>`, in JSX. Correct me if I'm wrong, but I think that would always blow up. If you're passing an actual React Component into `this.props.children` within `textComponent={this.props.children}` on the other hand, I guess that would work(?), but then you can end up having some people stuck with inconsistent / multiple dynamic renderings of the parent elements for `<Formatted*>` tags, spread out between different files within a scaled out project. Or another way of looking at it, it obfuscates the parent elements rendered for the `<Formatted*>` tags making it harder for a new developer within a project to see what's going on without digging into other files. It also makes the developer have to do a few extra acrobatics to make a different implementation of the `<Formatted*>` tags they want, whereas wrapping it inside a `React.Fragment` would simply allow the developer to just implement `<Formatted*>` on the exact same file as the Component they're using it in, exactly the way that's intuitive to them with the rest of React. A `React.Fragment` would allow new developers to just ignore the details of the implementation of `<IntlProvider>` to get something simple they want done right away within a larger project. People will also get a feel of `React.Fragment`s and the wonderfulness in potential that they can provide as First-Class Component citizens! =) That's my very long answer for a short question. =D Cheers!

Reply:
Sorry, my code above is wrong. This is the right one:
```jsx
<IntlProvider textComponent={props => props.children} />
```
This should always work both with strings (since react 16 supports rendering strings) and components.

Reply:
One weird thing `textComponent={props => props.children}` does is that it creates a `<textComponent>` Element inside the React Component Tree (inspecting with React Dev Tools) but **not** the actual rendered DOM itself. Fragments do not have this extra element in the React Component Tree, but I honestly don't know which one is more efficient in terms of processing. React-Intl usage of Context gives me a headache. =D Trying to think of use-cases for hijacking `textComponent` functionality using `props.children` with my own Components inside the children of `<IntlProvider>`... If there is one, would it make more sense to just create a dumb Component within the LanguageProvider folder and throw that into `textComponent` instead? Sorry, more questions than answers.